instances/cloudpak/cp4ba/postdeploy/ier-route.yaml

```diff
@@ -0,0 +1 @@
+# yaml file here for route (need to populate this with sh script or helm etc )
```

Comment:
Can you put the route spec here. Leave the value of the route to changeme or something. That way Jeff knows what to do then